gnucash/import-export/import-main-matcher.c

```diff
@@ -570,6 +591,9 @@ on_matcher_ok_clicked (GtkButton *button, GNCImportMainMatcher *info)
     }
     while (gtk_tree_model_iter_next (model, &iter));
 
+    DEBUG ("commit_edit");
+    g_list_free_full (accounts_modified, (GDestroyNotify)xaccAccountCommitEdit);
```

Comment:
Nice trick :wink:

Reply:
> It looks technically ok. I presume you actually tested the performance improvement is some way ?

Anecdotally yes seems faster; the real pain point was importing 6months of CSV in Windows.

Reply:
[Profiling](https://wiki.gnome.org/Apps/Sysprof) data would be better than "anecdotally seems faster". Even wall-clock timing the same import with and without the change would be better than "seems faster".